Makefile

```diff
@@ -82,6 +82,13 @@ verify-vendor:
 
 #===============================================================================
 
+.PHONY: bump-dependencies
+bump-dependencies:
+	dep ensure -update github.com/atlassian/smith
+	make update-vendor
```

Comment:
I think it'd be better to not call make from make. You can move the line above into a separate target and then change `bump-dependencies` to depend on both targets.

Reply:
Makefiles calling Makefiles are the path to insanity.

I worked at a place once where they used Makefiles to generate Makefiles. That might be the only thing worse than this.